source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -447,6 +460,38 @@ TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndCurrentLevel_Configures
   EXPECT_EQ(expected_output_function_value, actual_output_function_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, FetchMultiple_ReturnsResultOfExpectedSize)
+{
+  const char* channel_name = "0";
+  const int timeout = 5;
+  const int expected_count = 20;
+  ::grpc::ClientContext context;
+  dcpower::FetchMultipleRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
```

Comment:
Fetch multiple API implementation can be part of helper method. Given section of code body can include setting of attribute and initiate

Reply:
Great suggestion! Refactored test body to extract call to fetch multiple API into a function and now test is much more readable.